### ISSUE TITLE
Grafana eats CPU and memory on Chrome and FF

### ISSUE THREAD

**Issue by slayer** [grafana/grafana#11604]
Hi

I have problem with grafana prometheus dashboard, unexpectedly my dashboard became very slow and eats 100% of CPU and up to 2.5GB memory

### What Grafana version are you using?
 Grafana v5.0.4 (commit: 7dc36ae) - the latest docker image

### What datasource are you using?
prometheus

### What OS are you running grafana on?
Linux

### What did you do?
Try to load dashboard

### Include raw network request & response
![selection_052](https://user-images.githubusercontent.com/57521/38806689-c0ae59f2-4182-11e8-89bb-38bcecd67397.png)
![selection_051](https://user-images.githubusercontent.com/57521/38806694-c365351c-4182-11e8-838f-9e3b392ce427.png)

Chrome version 65.0.3325.181 (Official Build) (64-bit)
Firefox version 59.0.2 (64-bit)

**Comment by bergquist** [comment]
Weird. No similar reports (which would be expected if this was a common bug)

Could you please share your dashboard?

**Comment by torkelo** [comment]
and the number of data points returned by all your queries, maybe you have a long time range with small step parameter?

**Comment by slayer** [comment]
@daniellee yes, but it can measure time network requests or size results from first request and show warnings if the limits are exceeded and stops loading data again (if refresh is turned on).
Now it requests data again and again and there is no possibility to edit dashboard or turn refresh off because browser event loop is blocked.

**Comment by slayer** [comment]
I have an idea:  grafana can "yields" and give control to UI and that should helps
It can be done via periodical `setTimeout` - browser will give control to UI. 
We have done something similar on huge D3 graphs.

**Comment by jonathangaldino** [comment]
I have a **very** similar problem. Some info:

**What Grafana version are you using?**
Grafana 6.3.4

**What datasource are you using?**
MySQL

**What OS are you running grafana on?**
Windows 7 - SP1 x64

**What did you do?**
Just opnened the grafana server by double clicking grafana-server.exe

**If related to metric query / data viz:**
Not using metric or anything related.


I just opened the grafana-server.exe, nothing else. Started to monitoring the use of memor through window's task manager and I noticed that every *flick of update* the use of memory was getting higher and higher. Even with NOTHING getting used, no datasource, no dashboard, nothing. Just logged in.
